src/python/openvino_genai/__init__.pyi

```diff
@@ -0,0 +1,40 @@
+"""
+openvino genai module namespace, exposing pipelines and configs to create these pipelines.
+"""
+from __future__ import annotations
+import openvino as openvino
+from openvino_genai.py_openvino_genai import Adapter
+from openvino_genai.py_openvino_genai import AdapterConfig
+from openvino_genai.py_openvino_genai import AggregationMode
+from openvino_genai.py_openvino_genai import AutoencoderKL
+from openvino_genai.py_openvino_genai import CLIPTextModel
+from openvino_genai.py_openvino_genai import CLIPTextModelWithProjection
+from openvino_genai.py_openvino_genai import CacheEvictionConfig
+from openvino_genai.py_openvino_genai import ChunkStreamerBase
+from openvino_genai.py_openvino_genai import ContinuousBatchingPipeline
+from openvino_genai.py_openvino_genai import CppStdGenerator
+from openvino_genai.py_openvino_genai import DecodedResults
+from openvino_genai.py_openvino_genai import EncodedResults
+from openvino_genai.py_openvino_genai import GenerationConfig
+from openvino_genai.py_openvino_genai import GenerationResult
+from openvino_genai.py_openvino_genai import Generator
+from openvino_genai.py_openvino_genai import ImageGenerationConfig
+from openvino_genai.py_openvino_genai import LLMPipeline
+from openvino_genai.py_openvino_genai import PerfMetrics
+from openvino_genai.py_openvino_genai import RawPerfMetrics
+from openvino_genai.py_openvino_genai import Scheduler
+from openvino_genai.py_openvino_genai import SchedulerConfig
+from openvino_genai.py_openvino_genai import StopCriteria
+from openvino_genai.py_openvino_genai import StreamerBase
+from openvino_genai.py_openvino_genai import Text2ImagePipeline
+from openvino_genai.py_openvino_genai import TokenizedInputs
+from openvino_genai.py_openvino_genai import Tokenizer
+from openvino_genai.py_openvino_genai import UNet2DConditionModel
+from openvino_genai.py_openvino_genai import VLMPipeline
+from openvino_genai.py_openvino_genai import WhisperGenerationConfig
+from openvino_genai.py_openvino_genai import WhisperPipeline
+from openvino_genai.py_openvino_genai import draft_model
+import os as os
+from . import py_openvino_genai
+__all__ = ['Adapter', 'AdapterConfig', 'AggregationMode', 'AutoencoderKL', 'CLIPTextModel', 'CLIPTextModelWithProjection', 'CacheEvictionConfig', 'ChunkStreamerBase', 'ContinuousBatchingPipeline', 'CppStdGenerator', 'DecodedResults', 'EncodedResults', 'GenerationConfig', 'GenerationResult', 'Generator', 'ImageGenerationConfig', 'LLMPipeline', 'PerfMetrics', 'RawPerfMetrics', 'Scheduler', 'SchedulerConfig', 'StopCriteria', 'StreamerBase', 'Text2ImagePipeline', 'TokenizedInputs', 'Tokenizer', 'UNet2DConditionModel', 'VLMPipeline', 'WhisperGenerationConfig', 'WhisperPipeline', 'draft_model', 'openvino', 'os', 'py_openvino_genai']
```

Comment:
Looks like it is bybind11-stubgen bug, mypy-stubgen doesn't export os. 
Commited mypy-stubgen output for comparison: https://github.com/openvinotoolkit/openvino.genai/pull/1158/commits/1234257b4abca8c7159e2c123e6a1ca034c2e6da

Reply:
Looks like it's done because of `os.PathLike`

In general, it's not a problem as `os` is standard module. I think we can merge `pybind11-stubgen` output as start point.